frontend/src/utils/getRoutes.ts

```diff
@@ -32,6 +32,7 @@ export function getNextRoutes(src: string): string[] {
           .replace("/page.tsx", "")
           .replace(/\[locale\]/g, "")
           .replace(/\\/g, "/")
+          .replace(/\[id\]/g, "1") // for id-based routes like /opportunity/[id]
```

Comment:
allow pa11y to run on the opportunity listing page by just setting any `[id]` found to 1 - resulting in `/opportunity/1`